tests/memgraph/test_query_builder.py

```diff
@@ -1262,6 +1264,52 @@ def test_add_string_complete(memgraph):
     mock.assert_called_with(expected_query)
 
 
+def test_set_label(memgraph):
```

Comment:
Check out what you can do with SET clause and then maybe add a couple of more tests: https://memgraph.com/docs/cypher-manual/clauses/set